.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: minimal
 env: GO111MODULE=on
 git:
-  depth: 1
+  depth: false
 install:
   - eval "$(gimme)"
 stages:
-  - name: 'Lint markdown files'
```

Comment:
no real reason to specify 'name:' here so removing for consistency with other stages